data-pipeline-ffi/src/error.rs

```diff
@@ -57,6 +59,8 @@ impl Display for ExporterErrorCode {
             Self::NetworkUnknown => write!(f, "Unknown network error"),
             Self::Serde => write!(f, "Serialization/Deserialization error"),
             Self::TimedOut => write!(f, "Operation timed out"),
+            #[cfg(feature = "catch_panic")]
+            Self::Panic => write!(f, "Operation panicked"),
```

Comment:
Should we be returning more information here? Something that would be helpful when these errors wind up in telemetry logs?